### DISCUSSION TITLE
HALMET boards in production

### DISCUSSION THREAD

**Discussion by mairas** [hatlabs/discussions#80] [General]
I'm happy to inform I've just placed the initial production order of HALMET (Hat Labs Marine Engine & Tank interface). HALMET is a new board that combines the capabilities of the [SH-ESP32](https://shop.hatlabs.fi/products/sh-esp32) and the [Engine Top Hat](https://shop.hatlabs.fi/products/sh-esp32-engine-top-hat). It can be used to measure essential engine and tank parameters such as RPM, alarms, tank fill level and oil pressure for most available engines and tank sensors.

![HALMET_v1 0 0-a1_1200](https://github.com/hatlabs/discussions/assets/407010/12973c78-8a8e-4835-abce-392230b77d2c)
![HALMET_v1 0 0-a1_enclosure](https://github.com/hatlabs/discussions/assets/407010/53b2bbd5-7909-4426-8990-40f97378ff4c)

**Comment by big-col** [comment]
Brilliant news, when will they be available please?

**Comment by mairas** [comment]
Look what the courier just brought!

Hold your horses, though - it'll be a few days before everything is ready for sales.

![PXL_20231218_190005149 MP](https://github.com/hatlabs/discussions/assets/407010/61d925c5-7afd-4a2c-be9b-ff06c3a3748a)

**Reply by big-col** [reply]
Super excited!!

**Reply by psm12** [reply]
Yes please!
